### PR TITLE
feat: Show Info popup on tx

### PR DESCRIPTION
**Pull request by Tulun**
Story details: https://app.shortcut.com/uma-project/story/5750
Screen Shot 2022-06-09 at 8 00 22 AM

Copy to be provided by Korpi / Tim.

### CODE REVIEW

**Review by amateima — COMMENTED** [review]
Isn't the Info icon supposed to appear only on the Ongoing table?
Screenshot 2022-06-09 at 12 20 20


P.S I thought we were going to implement this UI
Screenshot 2022-06-09 at 12 15 16

File: src/views/Transactions/Transactions.tsx [hunk 1]
```diff
@@ -117,13 +126,13 @@ const Transactions = () => {
...
                   title="Ongoing"
-                  headers={headers}
+                  headers={createPendingHeaders(openInfoModal)}
                   rows={ongoingTx}
                 />
               ) : (
                 <MobileTransactionsTable
                   title="Ongoing"
-                  headers={mobileHeaders}
+                  headers={createPendingMobileHeaders(openInfoModal)}
```
[the remark this answers is not in the document]
Comment (Tulun) [reply src/views/Transactions/Transactions.tsx:135]: What do you mean?t his function returns the headers.

File: src/views/Transactions/TransactionsTable/createTransactionTableJSX.tsx [hunk 2]
```diff
@@ -251,3 +252,27 @@ export const headers: ICell[] = [
     cellClassName: "header-cell",
   },
 ];
+
+export function createPendingHeaders(
+  onClick: React.MouseEventHandler<SVGSVGElement>
+) {
+  const h = headers;
```
[the remark this answers is not in the document]
Comment (Tulun) [reply src/views/Transactions/TransactionsTable/createTransactionTableJSX.tsx:259]: Defined above this func.

File: src/views/Transactions/TransactionsTable/FillTxInfoModal.tsx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+          </Info>
+          <Info>
+            Quisque malesuada justo non ligula aliquam viverra. Duis
+            sollicitudin risus sit amet nulla bibendum fermentum. Nunc sem
+            lacus, porttitor et mattis sagittis, dignissim vel sem. Vestibulum
+            pretium sem purus, et suscipit mi condimentum sit amet. Pellentesque
+            volutpat lectus et efficitur auctor. Duis molestie orci sit amet
+            nulla pulvinar, non vehicula diam mattis. Lorem ipsum dolor sit
+            amet, consectetur adipiscing elit. Donec lacus mi, vehicula at nisi
+            sed, scelerisque accumsan velit. In eu libero nisi.
```
[the remark this answers is not in the document]
Comment (Tulun) [reply src/views/Transactions/TransactionsTable/FillTxInfoModal.tsx:39]: Yup.

**Comment by Tulun** [comment]
> Isn't the Info icon supposed to appear only on the Ongoing table? Screenshot 2022-06-09 at 12 20 20
> 
> P.S I thought we were going to implement this UI Screenshot 2022-06-09 at 12 15 16

That would be up to design and the PM. I have no idea if this is actually going into production or it's just an idea.

File: src/views/Transactions/Transactions.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (daywiss) [reply src/views/Transactions/Transactions.tsx:135]: sorry i see it already existed in the previous code

File: src/views/Transactions/TransactionsTable/createMobileTransactionTableJSX.tsx [hunk 4]
```diff
@@ -208,3 +208,28 @@ export const mobileHeaders: ICell[] = [
     cellClassName: "header-cell",
   },
 ];
+
+export function createPendingMobileHeaders(
+  onClick: React.MouseEventHandler<SVGSVGElement>
+) {
+  const mh = mobileHeaders;
```
[the remark this answers is not in the document]
Comment (daywiss) [reply src/views/Transactions/TransactionsTable/createMobileTransactionTableJSX.tsx]: i see now

**Review by daywiss — APPROVED** [review]

**Pull request merged**